src/app/savings/saving-account-actions/savings-account-transactions/savings-account-transactions.component.scss

```
@@ -27,3 +27,126 @@
+}
+
+/* Print styles */
+@media print {
+  .mat-stepper-horizontal-line,
+  mat-card-actions,
+  .mat-stepper-horizontal {
+    display: none !important;
+  }
+
+  .success-container {
+    page-break-after: always;
```

Stylelint (17.3.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)